suite-common/suite-types/src/device.ts

```diff
@@ -54,6 +54,7 @@ export interface ExtendedDevice {
         isRetrieving: boolean; // To prevent consequential call of the TrezorConnect.evoluGetNode(...)
         evoluKeys: EvoluKeys | undefined;
     };
+    delegatedKey?: string; // hex-encoded P-256 private key
```

Comment:
@peter-sanderson I think we talked about that this sould be put to the labeling redocuer, right? It shouldn't be at the device.